lib/helper/Playwright.js

```diff
@@ -35,7 +35,7 @@ let defaultSelectorEnginesInitialized = false;
 
 const popupStore = new Popup();
 const consoleLogStore = new Console();
-const availableBrowsers = ['chromium', 'webkit', 'firefox'];
+const availableBrowsers = ['chromium', 'webkit', 'firefox', '_electron'];
```

Comment:
```suggestion
const availableBrowsers = ['chromium', 'webkit', 'firefox', 'electron'];
```
Maybe let's just call it `electron`? Why not?

Reply:
Hey,
sorry I had a few busy weeks and it took me a while to finish this PR. Please have a look again, I have renamed the browser from `_electron` to `electron` and added some documentation as well as some other needed changes.

Reply:
Thanks @cbayer97 
This was a very hard week (month) for me so I just got here to look at updated PR. Everything looks good! :+1: